src/NUnitFramework/tests/Constraints/IndexerConstraintTests.cs

```diff
@@ -106,7 +126,7 @@ public void CanMatchShadowIndexer()
         }
 
         [Test]
-        public void CanMatchShadowNamedIndexer()
+        public void CanMatchHiddenNamedIndexer()
```

Comment:
These two tests are demonstrating matching the *hiding* members, not the *hidden* members, right?

Reply:
errr, yes :man_facepalming:

Reply:
This does sound strange - `CanMatchHidingNamedIndexer`. Does this make sense?